rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java

```diff
@@ -98,20 +99,32 @@ public static void main(String[] args) throws Exception {
 
     public static class EnrichmentFunction
             extends RichCoFlatMapFunction<TaxiRide, TaxiFare, RideAndFare> {
-
+        private ValueState<TaxiFare> fareState;
```

Comment:
should be `transient`
private transient
see details:
https://stackoverflow.com/a/72558397

Reply:
done